status-im/status-react
# Add screen for key-pairs and accounts inside wallet settings

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #19915 
fixes #19916 

### Summary

* This PR attempts to add the initial UI for the Wallet Settings - Key pairs and accounts screen
  * The initial includes the logic and view for displaying key pairs and accounts, but not logic for the managing the items.
    * This was done so we can start sharing the work related to managing key pairs/accounts and this screen is needed as a starting point for other features.
  * This PR also adds a feature flag for the wallet settings, which is disabled by default. This flag should allow us to merge this initial UI and other PRs safely without needing a complete design review. Eventually when we start releasing functionality we can remove the feature-flag or selectively choose what should still be feature flagged.

#### Platforms


- Android
- iOS

#### Areas that maybe impacted


##### Functional

- Wallet Settings
  - Key pairs and accounts
 
### Steps to test


- Open the status mobile app
- Open the user profile settings menu
- Ensure the feature-flag for `wallet-settings` is activated
- Ensure the feature-flag for `keypairs-and-accounts` is activated
- Open the wallet settings
- Open the "Key pairs and accounts" settings inside wallet settings
- Press on the action menu icon for a key-pair item



### Before and after screenshots comparison

#### After Changes - Initial UI for displaying key-pairs with accounts, along with opening the action-menu.

https://github.com/status-im/status-mobile/assets/2845768/ed7f27d1-acad-4367-bc49-5332918ec0c6

status: ready
